lib/model_renamer/rename.rb

```diff
@@ -21,29 +21,34 @@ def rename_and_generate_migrations
   end
 
   def rename
-    replace_all_occurrences
+    rename_files_and_directories
+    rename_in_files
   end
 
   def generate_migrations
     MigrationGenerator.new(@variations_generator.underscore_variations).create_migration_file
   end
 
-  private
+  def rename_files_and_directories
+    rename_files
+    rename_directories
+  end
 
-  def replace_all_occurrences
+  def rename_in_files
     all_filepaths.each do |filepath|
       replace_all_variations_in_file filepath
-      rename_file filepath
     end
   end
 
+  private
+
   def variation_pairs
     @variation_pairs ||= @variations_generator.pairs_to_convert
   end
 
   def all_filepaths
     Find.find('.').to_a.reject do |path|
-      FileTest.directory?(path) || !acceptable_filetype?(path) || ignore_file?(path)
+      FileTest.directory?(path) || !acceptable_filetype?(path) || ignore_file?(path) || !File.file?(path)
```

Comment:
What does Find return that passed both the not directory and acceptable filtype checks, but was still not a file?